### PR TITLE
Convert Double.NEGATIVE_INFINITY value on map to Double max negative value

### PR DESCRIPTION
**Pull request by ashangit**
Double.NEGATIVE_INFINITY leads to send events to ES with records_lag_max field equal to -Infinity
which leads to parsing exception on events with this value
Here we will update this value by max negative double value which will be indexable in ES

### CODE REVIEW

File: readers/common/src/main/java/com/criteo/hadoop/garmadon/protobuf/ProtoConcatenator.java [hunk 1]
```diff
@@ -91,7 +91,13 @@ protected ProtoConcatenator() {
                 return concatMap;
             },
             (entry, eventMap) -> {
-                eventMap.put(entry.getKey().getName(), getRealFieldValue(entry.getValue()));
+                Object value;
+                if ("DOUBLE".equals(entry.getKey().getType().name())) {
+                    value = Double.NEGATIVE_INFINITY == (Double) entry.getValue() ? -Double.MAX_VALUE : getRealFieldValue(entry.getValue());
```
Comment (mchataigner) [inline readers/common/src/main/java/com/criteo/hadoop/garmadon/protobuf/ProtoConcatenator.java]: Use `Double.MIN_VALUE` instead

File: readers/common/src/test/java/com/criteo/hadoop/garmadon/reader/GarmadonMessageTest.java [hunk 2]
```diff
@@ -70,4 +71,18 @@ public void testToMapNormalizedWithPortInfo() {
...
+        long timestampMillis = System.currentTimeMillis();
+        EventHeaderProtos.Header header = EventHeaderProtos.Header.newBuilder().build();
+        FlinkEventProtos.OperatorEvent.Builder operatorEvent = FlinkEventProtos.OperatorEvent.newBuilder()
+            .setRecordsLagMax(Double.NEGATIVE_INFINITY);
+
+        GarmadonMessage message = new GarmadonMessage(GarmadonSerialization.TypeMarker.FLINK_OPERATOR_EVENT, timestampMillis, header, operatorEvent.build(), null);
+
+        Map<String, Object> map = message.toMap(true, true);
+
+        TestCase.assertEquals(-Double.MAX_VALUE, map.get("records_lag_max"));
```
Comment (mchataigner) [inline readers/common/src/test/java/com/criteo/hadoop/garmadon/reader/GarmadonMessageTest.java]: Use `Double.MIN_VALUE` instead

File: readers/common/src/main/java/com/criteo/hadoop/garmadon/protobuf/ProtoConcatenator.java (hunk 1 above)
Comment (ashangit) [reply readers/common/src/main/java/com/criteo/hadoop/garmadon/protobuf/ProtoConcatenator.java]: For NEGATIVE_INFINITY is more closer to -Double.MAX_VALUE than Double.MIN_VALUE

File: readers/common/src/test/java/com/criteo/hadoop/garmadon/reader/GarmadonMessageTest.java (hunk 2 above)
Comment (ashangit) [reply readers/common/src/test/java/com/criteo/hadoop/garmadon/reader/GarmadonMessageTest.java]: idem

File: readers/common/src/main/java/com/criteo/hadoop/garmadon/protobuf/ProtoConcatenator.java (hunk 1 above)
Comment (mchataigner) [reply readers/common/src/main/java/com/criteo/hadoop/garmadon/protobuf/ProtoConcatenator.java]: -Double.MAX_VALUE is the correct value (as Double.MIN_VALUE is the smallest positive nonzero value of type double.

File: readers/common/src/test/java/com/criteo/hadoop/garmadon/reader/GarmadonMessageTest.java (hunk 2 above)
Comment (mchataigner) [reply readers/common/src/test/java/com/criteo/hadoop/garmadon/reader/GarmadonMessageTest.java]: -Double.MAX_VALUE is the correct value (as Double.MIN_VALUE is the smallest positive nonzero value of type double.

**Pull request merged**